### DISCUSSION TITLE
Documentation about installing a Skip Framework?

### DISCUSSION THREAD

**Discussion by sturmenta** [skiptools/skip#297] [Q&A]
I found some guidance on https://skip.tools/docs/gettingstarted/#separate-apps but I think should be useful to add little guide or something like that in https://skip.tools/docs/modules/

For example I want to use SkipKit and I do not where to start

**Comment by marcprux** [comment]
Do you mean something along the lines of “to add this module to your project, edit the Package.swift and add the following lines…”?

That would be useful. We'll look into doing that for the various Skip modules.
